.github/CHANGELOG.md

```diff
@@ -49,6 +49,9 @@
 
 ### Improvements
 
+* Update `lightning.tensor` python layer unit tests.
```

Comment:
```suggestion
* Update the `lightning.tensor` Python layer unit tests, as `lightning.tensor` cannot be cleaned up like other state-vector devices because the data is attached to the graph. It is recommended to use one device per circuit for `lightning.tensor`.
```